### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by asadmisrani**
All important files were created successfully.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1 @@
+.my-style { color: tomato; }
```
Comment (abdulhamiid) [inline style.css]: According to the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) please let's add a valid CSS color property to the `h1` tag.
`A .css file that includes one class with styles for your header (make it in your favorite color).` 

👇 This is not a valid color 😀, kindly fix
```
.my-style { color: tomato; }
```

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+Asad Shareef
+
+👤 Author
+
+GitHub: @asadmisrani
+Twitter: @asadshareef
+LinkedIn: /asadsahreef
+
+📝 License
+This project is MIT licensed.
```
Comment (abdulhamiid) [inline README.md]: - [ ] A README file is used to communicate important information about your project and also contains all the necessary information for developers to get started using and contributing to your project.

- [x] Please kindly note that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo to include all important information about your project 😉.

image

- [ ] The only optional section is the FAQ. Please kindly update your readme and customize all entries accordingly 👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @asadmisrani 👋,  good job working on setting up your first project at Microverse! 💯 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

**YOUR PULL REQUEST SUMMARY** 👇 

image

- Please kindly update your pull request summary 
  - Pull Request should have a descriptive title telling you what changes are inside.
  - Pull Request should have a summary describing introduced changes in more detail.
## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (asadmisrani) [reply README.md]: Hello @abdulhamiid,
I have make changes in CSS file with tomato color to red color, and I also made changes in and make as per requirements of Microverse in README file so you can check that again and kindly approve my exercise to go for further thank you.
Regards

Comment (abdulhamiid) [reply README.md]: Hi @asadmisrani, kindly submit your project on your dashboard for re-review.

**Comment by asadmisrani** [comment]
Hello  ***@***.*** *
*I have submitted my project in the dashboard kindly re-review my project.*
*Thank You*

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @asadmisrani , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**